src/PhpSpec/Message/CurrentExample.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Message;
+
+class CurrentExample
```

Comment:
Because this class is a simple value you can finalise it (although you'd have to change the tests to use a concrete value rather than a double).